hotxlfp/formulas/mathtrig.py

```diff
@@ -245,23 +245,36 @@ def SUMIF(args, criteria):
 
 @dispatcher.register_for("CEILING", "CEILING.MATH", "CEILING.PRECISE")
 def CEILING(number, significance=1):
-    number = utils.parse_number(number)
+    number = torch.tensor(utils.parse_number(number))
     significance = utils.parse_number(significance)
+    if not isinstance(significance, torch.Tensor) or significance.size(dim=0) == 1:
+        significance = torch.broadcast_to(torch.tensor(significance), number.size())
 
     if utils.any_is_error((number, significance)):
         return error.VALUE
-    if significance == 0:
-        return 0
+    if number.size(dim=0) != significance.size(dim=0):
+        return error.VALUE
 
-    positive_significance = significance > 0
-    significance = abs(significance)
```

Comment:
I see the old code is doing abs(significance) here. Do you need that in your code as well?

Reply:
thanks i fixed this